tests/api-suite-info/00-get-graph-raw-1/suite.rc

```diff
@@ -4,7 +4,10 @@
 [runtime]
     [[t1]]
         script = """
-ctb-get-graph-raw 'start_point_string=1' 'stop_point_string=1' 'group_nodes=T' \
+# Use the same Python (and lib) as the Cylc version being run.
+PYTHON=$(head -n 1 $(which cylc))
```

Comment:
Heck, why do we need this?

Reply:
Can't quite recall, sorry, damn it.  I guess because this involves invoking a Python script that uses the Cylc library without going through the Cylc CLI (so it's possible to use the wrong Python ...).

Reply:
Yep that'll be it. Ok, don't worry about it now, we should probably prevent this logic from proliferating though.